docs/AppendixA-FAQ.md

```diff
@@ -1 +1,34 @@
 # Appendix A - FAQ
+
+This section covers some of the most commonly encountered errors when running 
+the insertion scripts. They are divided into 3 sections depending on whether 
+the errors are related to the LORIS-MRI installation, LORIS front-end modules 
+setup, or LORIS-MRI scripts.
+
+The user is referred to the [README.md](../README.md) which includes a 
+post-installation verification section. **TODO: link to the proper sub-section.**
+
+|**Stage**  | **Error** | **Cause** | **How to Fix**|
+|:-----:|:------|:------|:----------| 
```

Comment:
lol I knew this was coming, but it is extremely hard to have the freedom in resizing tables in .md. But I will look into it, I definitely agree and knew this should be fixed :) You proposal of 3 tables, therefore getting rid of one column is awesome. It might be what I need actually!

Reply:
tried this in the upcoming commit. I also reduced the studyUID by deleting some numbers from it. Will see how this turns out to be :)